src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,42 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+
+        if (!(robot.getX() == toX) || !(robot.getY() == toY)) {
+
+            Direction robotDirection;
+
+            if (robot.getX() != toX) {
+                if (robot.getX() > toX) {
+                    robotDirection = Direction.LEFT;
```

Comment:
we can simplify it, doing smt like this: 
robotDirection = (robot.getX() < toX) ? Direction.RIGHT : Direction.LEFT;

Reply:
ok, i will try it.